src/PointwiseFunctions/GeneralRelativity/Psi4.hpp

```diff
@@ -0,0 +1,61 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+
+#include "DataStructures/DataBox/Tag.hpp"
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Domain/CoordinateMaps/Tags.hpp"
+#include "Domain/ElementMap.hpp"
+#include "Domain/Tags.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+
+/// \cond
+namespace gsl {
+template <typename>
+struct not_null;
+}  // namespace gsl
+/// \endcond
+
+namespace gr {
+
+/// @{
+/*!
+ * \ingroup GeneralRelativityGroup
+ * \brief Computes Newman Penrose quantity $\Psi_4$ using the characteristic
+ * field U\f$^{8+}\f$ and complex vector \f$\bar{m}^i\f$.
+ *
+ * \details Computes $\Psi_4$ as: \f$\Psi_4\f$ =
```

Comment:
Why jump in and out of math mode here instead of doing the entire equation in math mode?